src/io/bt.js

```diff
@@ -128,6 +128,16 @@ class BT extends JSONRPC {
                 window.clearTimeout(this._discoverTimeoutID);
             }
             break;
+        //TODO why isn't 'user did pick peripheral' here?
```

Comment:
Oops - there definitely should be a 'userDidPickPeripheral' handler here! Connecting on BT using CDM doesn't work for me currently.